src/main/java/aquality/selenium/core/localization/ILocalizationModule.java

```diff
@@ -0,0 +1,11 @@
+package aquality.selenium.core.localization;
+
+public interface ILocalizationModule {
```

Comment:
Add please documentation for class and methods

Reply:
done